<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;ui/binkaibulet.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(&quot;icons/icon.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 30px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Janis Kendaraan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: gold;</string>
     </property>
     <property name="text">
      <string>Data Pelayanan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Kode Pelayanan</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="gridPelayanan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>181</y>
       <width>541</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
gridline-color: rgb(85, 255, 0);</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: gold;
color: rgb(85, 255, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtKodePelayanan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCari">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
      </property>
      <property name="text">
       <string>cari</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
color: rgb(85, 255, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnClear">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHapus">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSimpan">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
      </property>
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="JenisKendaraan">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sepeda Motor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mobil</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: gold;
color: rgb(85, 255, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="ServiceBesar">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
      <property name="text">
       <string>Service Besar</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ServiceRingan">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
      <property name="text">
       <string>Service Ringan</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Jasa Pelayanan</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: gold;
color: rgb(85, 255, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnCariIdService">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
      </property>
      <property name="text">
       <string>cari</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIDSERVICE">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
      <property name="placeholderText">
       <string>ID Service</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtJenisService">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>ID Service</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 130px;
color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: gold;</string>
     </property>
     <property name="text">
      <string>Bengkel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: gold;</string>
     </property>
     <property name="text">
      <string>Kendaraan</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;icons/icon.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame</zorder>
   <zorder>frame_9</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
